fundamentals/streaming/SynchronousWithNewtonsoftJson/.gitignore

```diff
@@ -0,0 +1,350 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Can this be merged into the gitignore in the repo root?

Reply:
I'm guessing it was a copy/paste oversight. Please remove your .gitignore as we have one.

Reply:
The parent `.gitignore` ignores the `Properties` directory, excluding the `launchSettings.json` file.

Reply:
Oh, but, yes, the individual project doesn't need its own. :-)